Sources/Core/Microsoft.StreamProcessing/Egress/Temporal/TemporalEgressObservable.cs

```diff
@@ -41,9 +41,11 @@ public IDisposable Subscribe(IObserver<StreamEvent<TPayload>> observer)
             if (!Config.ForceRowBasedExecution && this.source.Properties.IsColumnar && typeof(TPayload).CanRepresentAsColumnar() && CanGenerateColumnar())
                 pipe = GetPipe(observer);
             else
+            {
                 pipe = new StreamEventEgressPipe<TPayload>(
-                observer,
-                this.container);
+                    observer,
+                    this.container);
+            }
```

Comment:
Both the if and else clauses here just assign a value to pipe. Perhaps we should instead convert to a conditional assignment pipe = … ? … : …